pegasus/helpers/section_api_helpers.rb

```diff
@@ -6,6 +6,15 @@ def self.admin?(user_id)
   end
 end
 
+class DashboardUser
+  def self.to_self_view(user_row)
```

Comment:
[Hash#slice](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/hash/slice.rb#L21-L24) looks to be a Rails-only extension.